server/routes/functionalSkills/functionalSkillsController.ts

```diff
@@ -14,18 +18,50 @@ export default class FunctionalSkillsController {
       prisonNumber,
       req.user.username,
     )
-    const latestFunctionalSkillsFromCurious = mostRecentFunctionalSkills(functionalSkillsFromCurious)
-    const allFunctionalSkillsFromCurious = allFunctionalSkills(functionalSkillsFromCurious)
 
     const { problemRetrievingData } = functionalSkillsFromCurious
+
+    // Get all functional skills with the type 'ENGLISH' and add the prison name to the object for each from the prison service
     const englishSkills = functionalSkillsByType(functionalSkillsFromCurious.assessments, 'ENGLISH')
+    await Promise.all(
```

Comment:
We've used this pattern before to update the prison names in controllers (without the use of a service), I think it looks quite ugly and repetitive here but couldn't think of another way to do it!

We may need to come up with a pattern in the future for the wider code base as this is the 2nd time we've used this. (Plus the eslint error for no-param-reassign again)